appveyor/README.md

```diff
@@ -44,3 +16,18 @@ optimize downloads and caching for Appveyor.com CI builds.
     * protoc 2.5.1
     * Boost 1.67
     * OpenSSL 1.0.2j (64-bit) from http://slproweb.com/products/Win32OpenSSL.html
+* Version 17.05 splits the single zip file into several to stay under Github's
+  file size limits.
+  * Built using and for Visual Studio 2017 with prebuilt OpenSSL
+  * `rippled_deps.protoc.2.5.1.zip`
```

Comment:
pretty sure we don't need protoc anymore - we build a newer version anyhow as an EP in the cmake file

Reply:
That's easy enough to test!

Reply:
With the PROTOC dependency removed from the script, and the cache invalidated, Appveyor timed out, but not until after the build was done, so this is safe to remove.